tests/unit/test_log.py

```diff
@@ -27,14 +27,14 @@ def test_formatter_correctly_set(self):
 
 
 class TestSetFileHandler(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.logger = log.get_logger("s3_upload", log_level=logging.INFO)
-        log.set_file_handler(cls.logger, Path(__file__).parent)
-        cls.logger.setLevel(5)
-
-    @classmethod
-    def tearDownClass(cls):
+    def setUp(self):
```

Comment:
I was wondering why you changed from using a class method to instance method here?

Reply:
Where are they called in the tests?
<!-- Reviewable comment -OBewmvg39PhYvzKqMa3:bbl9r5f -->
<!-- Sent from Reviewable.io -->